src/content/self-hosted/administration/configuration.mdx

```diff
@@ -936,6 +936,15 @@ injectDevelopmentBinaries:
   enabled: true
 ```
 
+### insights
+
+Configures [Okteto Insights](administration/okteto-insights.mdx) feature in your Okteto instance.
+
+- `enabled`: Whether to enable Okteto Insights. Defaults to `false`.
+- `bearerSecret.name`: Name of the secret where the token to acces Okteto Insights metrics is stored. Defaults to `okteto-insights`.
+Default to `okteto-insights`.
```

Comment:
```suggestion
```

This appears potentially like an accidental duplicate. Disregard this suggestion if that's wrong 😄